media/css/mozorg/about.scss

```diff
@@ -126,33 +126,15 @@ $city-image-size: 106px;
     &.c-city-toronto::before {
         background-image: url('/media/img/mozorg/about/city-toronto.png');
     }
```

Comment:
We typically like to keep image files and folders organized by pages in which we use them in. This can definitely be something we can bring up for the website refresh we're working and planning on.
We also use these images in https://www.mozilla.org/en-US/careers/locations/, so I can have a chat with the team on having some universal image folders we could use throughout the site